tests/test_session.py

```diff
@@ -199,6 +200,61 @@ def test_session_connect_ssh_userauth_password_error(monkeypatch, patched_sessio
     assert session._channel is None
     fake_ssh_free.assert_called_once_with("<session object>")
 
+def _fake_ssh_pki_import_privkey_file(filename, passphrase, _a, _b, pkey):
+    pkey[0] = "<key object from {} with pass {}>".format(filename, passphrase)
+    import pystassh.api
```

Comment:
I can't remember why I added so many lazy imports in the tests file... It must be due to a compatibility issue with pytest at the time. I'll remove them in a separate commit!

Reply:
Initially I thought that those were there because there was a magic monkey patching before which I couldn't find. To be on the safe side I just followed the style of the rest of the file.

Reply:
I wrote this package 5 years ago, don't feel bad for refactoring stuff :smile: